src/Management/test/Endpoint.Test/CorsPolicyTest.cs

```diff
@@ -195,4 +196,39 @@ public async Task ConfiguresDefaultActuatorsCorsPolicyForGetRequestOnCloudFoundr
         response.Headers.Should().ContainKey("Access-Control-Allow-Origin");
         response.Headers.GetValues("Access-Control-Allow-Origin").Should().HaveCount(1).And.Contain("*");
     }
+
+    [Fact]
+    public async Task DefaultActuatorsCorsPolicyFiresBeforeCloudFoundrySecurity()
+    {
+        using var scope = new EnvironmentVariableScope("VCAP_APPLICATION", "{}");
+
+        WebApplicationBuilder builder = TestWebApplicationBuilderFactory.Create();
+        builder.Services.AddCloudFoundryActuator();
+        await using WebApplication app = builder.Build();
+        await app.StartAsync();
+        using HttpClient httpClient = app.GetTestClient();
+        var corsRequest = new HttpRequestMessage(HttpMethod.Options, new Uri("http://localhost/cloudfoundryapplication"));
+        corsRequest.Headers.Add("access-control-request-method", "GET");
+        corsRequest.Headers.Add("access-control-request-headers", "authorization");
+        corsRequest.Headers.Add("origin", "http://example.api.com");
+        HttpResponseMessage corsResponse = await httpClient.SendAsync(corsRequest);
+
+        corsResponse.StatusCode.Should().Be(HttpStatusCode.NoContent);
+        corsResponse.Headers.Should().ContainKey("Access-Control-Allow-Origin");
+        corsResponse.Headers.GetValues("Access-Control-Allow-Origin").Should().HaveCount(1).And.Contain("*");
+        corsResponse.Headers.Should().ContainKey("Access-Control-Allow-Headers");
+
+        corsResponse.Headers.GetValues("Access-Control-Allow-Headers").Should().HaveCount(1).And
+            .Contain("Authorization,X-Cf-App-Instance,Content-Type,Content-Disposition");
+
+        corsResponse.Headers.Should().ContainKey("Access-Control-Allow-Methods");
+        corsResponse.Headers.GetValues("Access-Control-Allow-Methods").Should().HaveCount(1).And.Contain("GET");
+
+        var request = new HttpRequestMessage(HttpMethod.Get, new Uri("http://localhost/cloudfoundryapplication"));
+        request.Headers.Add("Origin", "http://example.api.com");
+        HttpResponseMessage response = await httpClient.SendAsync(request);
```

Comment:
```suggestion
        var actuatorRequest = new HttpRequestMessage(HttpMethod.Get, new Uri("http://localhost/cloudfoundryapplication"));
        actuatorRequest.Headers.Add("Origin", "http://example.api.com");
        HttpResponseMessage response = await httpClient.SendAsync(actuatorRequest);
```
If there's only one request, the name `request` is fine. But if there are multiple, it should be clear from their names how they relate. Having both `somethingRequest` and `request` variables in scope makes it harder to track what's going on.